src/test/java/nl/tudelft/goalkeeper/util/console/ConsoleTest.java

```diff
@@ -100,4 +100,44 @@ void printlnColorEnabledTest() throws UnsupportedEncodingException {
         Console.println("das", ConsoleColor.CYAN);
         assertThat(out.toString(UTF8)).isEqualTo("\u001B[36mdas" + NEWLINE);
     }
+
+    /**
+     * Checks that printing an object works.
+     */
+    @Test
+    void printObjectTest() throws UnsupportedEncodingException {
+        Console.print(1);
+        assertThat(out.toString(UTF8)).isEqualTo("1");
+    }
+
+    /**
+     * Checks that printing an object works with colors.
+     */
+    @Test
+    void printColorObjectTest() throws UnsupportedEncodingException {
+        Console.setUseColor(true);
+        Console.print(-1, ConsoleColor.RED);
+        assertThat(out.toString(UTF8)).isEqualTo(
+                ConsoleColor.RED.getAnsi() + "-1");
+    }
+
+    /**
+     * Checks that printing an object and appending a newline works.
+     */
+    @Test
+    void printlnObjectTest() throws UnsupportedEncodingException {
+        Console.println(0.5f);
+        assertThat(out.toString(UTF8)).isEqualTo("0.5" + NEWLINE);
+    }
+
+    /**
+     * Checks that printing an object and appending a newline works with colors.
+     */
+    @Test
+    void printlnColorObjectTest() throws UnsupportedEncodingException {
+        Console.setUseColor(true);
+        Console.println(-0.5, ConsoleColor.WHITE);
+        assertThat(out.toString(UTF8)).isEqualTo(
+                ConsoleColor.WHITE.getAnsi() + "-0.5" + NEWLINE);
+    }
```

Comment:
Have you thought about the `NULL` case as this is treated differently between `String.ValueOf()` and `Obj.toString`?

Reply:
Yes, which is exactly why I used `String.valueOf()` just like `System.out` and `StringBuilder` use. I'd rather have `null` be printed than exceptions being thrown which happen with `.toString()`.

Reply:
Alright sounds good to me! Just making sure the difference was accounted for!